lib/documents/schemas/export_health_certificates.json

```diff
@@ -1485,7 +1485,7 @@
         {"label": "Eswatini (previously Swaziland)", "value": "eswatini-previously-swaziland"},
         {"label": "Ethiopia", "value": "ethiopia"},
         {"label": "EU", "value": "eu"},
-        {"label": "Eurasian Customs Union", "value": "eurasian-customs-union"},
+        {"label": "Eurasian Economic Union", "value": "eurasian-customs-union"},
```

Comment:
Do we need to change the value to `eurasian-economic-union`?

Reply:
If we change the value we will have to create a rake task to change all the old certificates to have the value ```eurasian-economic-union```. This is a quick fix so the users will see Eurasian Economic Union on the frontend while the backend can remain the same.

Reply:
Cool, just wanted to confirm 👍 Looks good